ui/src/App/Main.jsx

```diff
@@ -44,6 +44,9 @@ import isLoggedInQuery from './isLoggedIn.graphql';
     ':not(pre) > code': {
       ...theme.mixins.highlight,
     },
+    '.d2h-file-header': {
+      display: 'none',
+    },
```

Comment:
Instead of

![Screen Shot 2019-09-16 at 10 01 51 AM](https://user-images.githubusercontent.com/3766511/64964651-79509280-d869-11e9-9618-38b5d7c5585d.png)

Let's just show the diff:

![Screen Shot 2019-09-16 at 10 01 33 AM](https://user-images.githubusercontent.com/3766511/64964654-7c4b8300-d869-11e9-85d3-d76de412d8fb.png)

Reply:
OK! I'll update soon, thanks for the review :)

Reply:
@helfi92 Updated!

Reply:
Did something change in the styling. Clicking "View Diff" without changing the text shows:

![Screen Shot 2019-09-18 at 4 55 31 PM](https://user-images.githubusercontent.com/3766511/65185258-2aefef00-da35-11e9-80f8-d183444b219a.png)

Reply:
OK, I'll create a default view if the text has no diff. Do I need to add some default message in it? Like `The text is not changed` or something?

Reply:
>  Do I need to add some default message in it?

Leaving it empty should be fine.

Reply:
Let me know when this is ready for review :)

Reply:
@helfi92 , I just updated the PR and it's ready for review, thanks :)